src/components/Card.js

```diff
@@ -4,18 +4,34 @@ import emoji from 'emoji-dictionary';
 
 import './Card.css';
 
+
 class Card extends Component {
+  constructor(props){
+    super(props)
+
+  }
+
+
   render() {
+    const deleteCard = () => {
+        console.log(`This is the index from card props: ${this.props.index}`);
+        console.log(`This is the id from card props: ${this.props.id}`);
+        this.props.onDeleteCallback(this.props.index, this.props.id);
+      }
```

Comment:
It might make more sense to define this function `deleteCard` outside of `render` but still within `Card`. Then, to reference it, you would use `this.deleteCard`